rd_ui/app/views/query.html

```diff
@@ -110,7 +110,7 @@ <h4 class="modal-title">Query Archive</h4>
               <span class="text-muted">Data Source</span>
               <select ng-disabled="!isQueryOwner" ng-model="query.data_source_id" ng-change="updateDataSource()"
                       ng-options="ds.id as ds.name for ds in dataSources"></select>
-
+              <a ng-href={{dataSource.doc_url}}>{{dataSource.type_name}} documentation</a>
```

Comment:
![screenshot from 2016-11-11 11-38-55](https://cloud.githubusercontent.com/assets/947487/20240160/2fca6c4a-a8d7-11e6-9d75-f5701bb2f4d5.png)

Fixed so that it should only show when doc_url is present.